src/cli/utils/fileSystem.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from "fs";
+import { FileStatus } from "../../types/exercises.types";
+
+/**
+ * Returns status of file with name starting with `partialFileName` in `path`
+ * @param path Path to look into
+ * @param partialFileName Partial name to match
+ */
+export const isFilePresent = (
+    path: string,
+    partialFileName: string,
+): FileStatus => {
+    try {
+        const files = fs
+            .readdirSync(path)
+            .map((file) => file.startsWith(partialFileName));
+        if (files.length > 0) {
+            if (files.length === 1) {
+                return FileStatus.SUCCESS;
+            }
+            return FileStatus.INCONCLUSIVE;
+        }
+        // Should never reach here
+        return FileStatus.FAILURE;
```

Comment:
I think this is incorrect, because `INCONCLUSIVE` differs from `FAILURE`. Failure implies that the file couldn't be found, however inconclusive implies file could not be uniquely found.

Reply:
Yes, but you will never reach FAILURE in the chunk of code so what's the point of having it anyway